test/convenience-method-test.js

```diff
@@ -167,6 +167,25 @@ module.exports = {
           }, 1000);
         }*/
   },
+  ".scheduleJob({...}, fn)": {
+    "Runs job at interval based on object, repeating indefinitely, with 5 second variation": function(test) {
+      test.expect(4);
```

Comment:
No, only the invocations for 15,30,45 & 60 seconds will run and the last one will be between 55 & 65 seconds into the run, the 5th invocation would only run between 70 & 80 seconds into the run. The 65250 timeout should limit it to 4 invocations.